rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyParserVisitor.java

```diff
@@ -604,50 +579,39 @@ null, emptyList(),
         }
 
         @Override
-        public void visitConstructor(ConstructorNode ctor) {
+        public void visitConstructor(ConstructorNode constructor_) {
```

Comment:
Usually we reserve the `_` suffix when it concerns a Java keyword; constructor is not a keyword; what made you add that suffix here?

Reply:
I really don't know